iron-request.html

```diff
@@ -425,7 +425,9 @@
     _wwwFormUrlEncodePiece: function(str) {
       // Spec says to normalize newlines to \r\n and replace %20 spaces with +.
       // jQuery does this as well, so this is likely to be widely compatible.
-      if (str === null) {
+      try {
+        str.toString
```

Comment:
Is it sufficient to test for `str == null` (loose equality), and blame the user for bad behavior if she passes in `NaN` or something else weird?